core/controllers/practice_sessions.py

```diff
@@ -52,16 +54,22 @@ def get(self, topic_name):
         # Topic cannot be None as an exception will be thrown from its decorator
         # if so.
         topic = topic_fetchers.get_topic_by_name(topic_name)
+        subtopic_titles_list = self.request.get('selected_subtopics')
+        selected_subtopic_titles = json.loads(subtopic_titles_list)
+        selected_skill_ids = []
+        for subtopic in topic.subtopics:
+            if subtopic.title in selected_subtopic_titles:
+                selected_skill_ids.extend(subtopic.skill_ids)
```

Comment:
Else raise 400 (if there's some invalid item in the list)? Or should we not do that?

Reply:
I think it's fine to just ignore those? If some subtopics were deleted after the learner page was loaded, then this would throw an error, but it shouldn't, right?

Reply:
OK, sounds reasonable! Make sure you have a backend test for this case. Also, here, perhaps add a comment with some explanation of the desired behaviour and the rationale for it.